cmd/pdptool/main.go

```diff
@@ -885,6 +886,10 @@ var uploadFileCmd = &cli.Command{
 	},
 }
 
+// BREAKING CHANGE: createDataSet has been removed in PDP v2.2.0
+// Datasets are now created implicitly when adding the first piece via addPieces
+// To create a dataset, use the add-pieces command with your first piece
+/* DISABLED - createDataSetCmd removed in PDP v2.2.0
 var createDataSetCmd = &cli.Command{
```

Comment:
We should probably update this command instead of deleting it as there is no way to create dataSet now

Reply:
Makes sense, I have updated the createDataSetCmd in https://github.com/filecoin-project/curio/pull/685/commits/7666fb428d8c4f216a96ea8976a417f3bdef3f02.